A-基础教程/A2-神经网络基本原理/en-us/Step1 - BasicKnowledge/00 前言.md

```diff
@@ -1,23 +1,21 @@
 <!--Copyright © Microsoft Corporation. All rights reserved.
   适用于[License](https://github.com/Microsoft/ai-edu/blob/master/LICENSE.md)版权许可-->
 
-# 前言
+# Preface
 
-**What I cannot create, I do not understand.**
+**"What I cannot create, I do not understand." (American physicist Richard Feynman)**
 
-**如果我不能亲手搭建起来一个东西，那么我就不能理解它。 -- 美国物理学家理查德·费曼**
+In today's technologically advanced world, a lot of knowledge can be found on the Internet. However, the quality of blogs/articles is questionable, its focus is not clear, or it is directly copied from other people’s blogs. This situation makes it difficult for most beginners to learn. They may even be led astray, increasing the steepness of the learning curve. Of course, there are creators who are very responsible. The quality of their articles may be very high, but its scope is not enough. When you are enjoying yourself, there are no follow-up chapters and so it cannot be a good learning system.
 
-在互联网发达的今天，很多知识都可以从网络上找到，但是网络上的博客、文章的质量参差不齐，或者重点不明确，或者直接把别人的博客抄袭过来。这种状况使得广大的初学者们学习起来很困难，甚至误入歧途，增加了学习曲线的陡峭程度。当然也有很多博主非常非常负责任，文章质量很高，只是连续度不够，正看得过瘾的时候，没有后续章节了，无法形成知识体系。
+Beginners can choose to read textbooks or books on theory, but the chicken and the egg problem arises: If you don’t understand, then after reading you still won’t understand the theory; if you understand, then you don’t need to read the theory. This is a shortcoming of some textbooks and theoretical books.
 
-初学者也可以选择看一些教材或者理论书籍，但是，一个鸡生蛋蛋生鸡的问题出现了：如果你不懂，那么看完了理论你还是不会懂；如果你懂了，那么你就没必要看理论。这也是很多教材或者理论书籍的缺憾。
+The author has seen professor Andrew Ng's (Wu Enda) classes. The theoretical knowledge is simple yet deep, and is very clear. Although there are very few code examples, I still strongly recommend looking into Andrew Ng's classes. The author's experience is that Andrew Ng's videos can be saved on a mobile device, and you can watch and learn on your own time.
```

Comment:
Why do you mention this source and do not add a hyperlink so people know what you are talking about?